### PR TITLE
i18n(fr): add migrate-to-astro/from-gatsby.mdx

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

* Add French translation for [Migrating from Gatsby](https://docs.astro.build/en/guides/migrate-to-astro/from-gatsby/) (`migrate-to-astro/from-gatsby.mdx`) added in #2232

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+
+- Article de blog : [Mon passage de Gatsby à Astro](https://www.joshfinnie.com/blog/my-switch-from-gatsby-to-astro/) (en).
+
+- Article de blog : [Pourquoi je suis passé de Gatsby à Astro](https://dev.to/askrodney/why-i-moved-to-astro-from-gatsby-3fck) (en).
+
+- Article de blog : [Migrer mon site web de Gatsby vers Astro](https://dev.to/flashblaze/migrating-my-website-from-gatsby-to-astro-2ej5) (en).
+
+- Article de blog : [Une autre migration : de Gatsby à Astro](https://logarithmicspirals.com/blog/migrating-from-gatsby-to-astro/) (en).
+
+- Article de blog : [Pourquoi et comment j'ai déplacé mon blog de Gatsby et React vers Astro et Preact](https://www.helmerdavila.com/blog/en/why-and-how-i-moved-my-blog-away-from-gatsby-and-react-to-astro-js-and-preact) (en).
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:581]:
```suggestion
- Article de blog : [Pourquoi et comment j'ai déplacé mon blog de Gatsby et React vers Astro et Preact](https://www.helmerdavila.com/blog/fr/jai-migre-mon-blog-de-gatsby-et-react-a-astrojs-et-preact).
```

Article available in French 🙌

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+
+:::note[Vous avez une ressource à partager ?]
+Si vous trouvez (ou créez !) une vidéo ou un article de blog utile sur la conversion d'un site Gatsby vers Astro, [éditez cette page](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx) et ajoutez-le ici !
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:584]:
```suggestion
Si vous trouvez (ou créez !) une vidéo ou un article de blog utile sur la conversion d'un site Gatsby vers Astro, [éditez cette page](https://github.com/withastro/docs/edit/main/src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx) et ajoutez-le ici !
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Some little editions, for the rest seem LGTM!

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+
+### Installer des intégrations (facultatif)
+
+Vous trouverez peut-être utile d'installer certaines des [intégrations facultatives d'Astro](/fr/guides/integrations-guide/) à utiliser lors de la conversion de votre projet Gatsby en Astro :
+
+- **@astrojs/react**: pour réutiliser certains de vos composants React orientés UI dans votre nouveau site Astro ou pour continuer à coder des composants React.
+
+- **@astrojs/mdx**: pour importer des fichiers MDX existants de votre projet Gatsby ou pour utiliser MDX dans votre nouveau site Astro.
+
+### Placer votre code dans `src`
```
[the remark this answers is not in the document]
Comment (ArmandPhilippot) [reply src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:93]: @thomasbnt Using the infinitive form make the guide consistent with the others (e.g. https://docs.astro.build/fr/guides/migrate-to-astro/from-create-react-app/#ajouter-votre-cra-%C3%A0-astro ). But if you think it's better, I'll change!

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx (hunk 1 above)
Comment (ArmandPhilippot) [reply src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:581]: Good catch! I'm so unaccustomed to seeing articles also available in French that I forgot to check...

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx (hunk 2 above)
Comment (ArmandPhilippot) [reply src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:584]: @thomasbnt This one I hesitated but the contribution guide explains that all changes must be done on the English pages. Then we update the translations, not the inverse. What do you think? Should I update?

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:93]: @ArmandPhilippot we can put to infinitive form, avoid this review for line 93 and 85, but change it for https://github.com/withastro/docs/pull/8762/files#diff-6cfd0ab819b7666ee1adf7c8616ee215e528d4c4482417d581931c05244832f7R43

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx [hunk 4]
```diff
@@ -0,0 +1,585 @@
...
+
+- [Les composants `.astro`](/fr/basics/astro-components/) ne sont pas écrits sous forme de fonctions exportées renvoyant des modèles de page. Au lieu de cela, vous diviserez votre code en une « barrière de code » pour votre JavaScript et en un corps dédié au HTML que vous générez.
+
+- [Données de fichiers locaux](/fr/guides/imports/): Gatsby utilise GraphQL pour récupérer des données à partir de vos fichiers de projet. Astro utilise les importations ESM et les fonctions d'attente de niveau supérieur (par exemple, [`Astro.glob()`](/fr/guides/imports/#astroglob), [`getCollection()`](/fr/guides/content-collections/#interroger-les-collections)) pour importer des données à partir de vos fichiers de projet. Vous pouvez ajouter manuellement GraphQL à votre projet Astro mais il n'est pas inclus par défaut.
+
+## Convertir votre projet Gatsby
+
+Chaque migration de projet sera différente, mais vous effectuerez certaines actions courantes lors de la conversion de Gatsby vers Astro.
+
+### Créez un nouveau projet Astro
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:43]:
```suggestion
### Créer un nouveau projet Astro
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]

File: src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx (hunk 2 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/migrate-to-astro/from-gatsby.mdx:584]: > @thomasbnt This one I hesitated but the contribution guide explains that all changes must be done on the English pages. Then we update the translations, not the inverse. What do you think? Should I update?

oooh yeah, you right! sorry!

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by dreyfus92 — APPROVED** [review]
Merci les gars, LGTM ✅

**Pull request merged**